net/vncrepeater/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2006-2014 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=vncrepeater
+PKG_VERSION:=0.14
+PKG_RELEASE:=1
+PKG_MAINTAINER:=JimmyS83 
+PKG_LICENSE:=GNU GPL
```

Comment:
Two minor comments: 

* You should check the license code from the SPDX list and use the corerct GNU GPL abbreviation depending on the upstream license, e.g. GPL-2.0
See  https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#common-license-tags-short-list